src/views/partials/doc-resource-weapon-properties.ejs

```diff
@@ -38,8 +38,8 @@
     </tr>
     <tr>
       <td align="left">desc</td>
-      <td align="left">The description of the weapon property.</td>
-      <td align="left">array</td>
+      <td align="left">Descriptions of the weapon property.</td>
```

Comment:
It's a singular description. Each element is a separate paragraph of the description.

Reply:
Gotcha, will update accordingly 👍